.env.dist

```diff
@@ -3,6 +3,7 @@ CHECKER_MEMBERSHIP_API_ENDPOINT=https://tue-lookup.test.gewis.nl/user/
 CHECKER_MEMBERSHIP_API_KEY=c2VjcmV0
 CHECKER_MEMBERSHIP_API_MAX_TOTAL_REQUESTS=200
 CHECKER_MEMBERSHIP_API_MAX_MANUAL_REQUESTS=20
+DOCTRINE_ROLE=gewisdb
```

Comment:
What happens here is that we allow a set of credentials for default and for reportdb: `DOCTRINE_DEFAULT_USER`/`DOCTRINE_REPORT_USER` and `DOCTRINE_DEFAULT_PASSWORD`/`DOCTRINE_REPORT_PASSWORD`

To me it make sense to implement the role on a database-level as well or make the credentials a application-level variable as well. In principle, the credentials determine what roles are available.

Reply:
What do you think about the solution I propose in https://github.com/tomudding/gewisdb/pull/2

Reply:
Thank you, I have inlined your changes into f80419ffa4270b16b0e7bf2aecd5c4384a279a21.